.babelrc

```diff
@@ -1,3 +1,6 @@
 {
-  "presets": ["es2015"]
+    "presets": ["es2015"],
+    "plugins": [
+        "transform-async-to-generator",
+    ],
```

Comment:
I assume babel loads this as a JS object rather than a JSON file, which is why these trailing commas work?